app/task_routes.py

```diff
@@ -0,0 +1,124 @@
+from app import db
+from app.models.task import Task
+import requests
+from flask import Blueprint, jsonify, request
+from sqlalchemy import desc
+from datetime import date
+import os
+
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+@tasks_bp.route("", methods=["GET"])
+def read_tasks():
+    sort_query = request.args.get("sort")
+    if sort_query:
+        if sort_query == "asc":
+            tasks = Task.query.order_by(Task.title)
+        else:
+            tasks = Task.query.order_by(desc(Task.title))
+    else:
+        tasks = Task.query.all()
```

Comment:
Nesting the conditionals isn't necessary for this scenario and actually doesn't work like expected. I can set the sort_query to any alphanumeric character or word and still receive the list of tasks in descending order. Here is an example of setting the sort query to hamster: https://alf-task-list-api.herokuapp.com/tasks?sort=hamster.

Reply:
If we want our data to be sorted in a specific way, given a specific query/command it is better to explicitly code that logic rather than having it be used in a 'catch all' scenario.

Reply:
lol and also duh !! Noted and refactored !